kernel_gateway/services/cell/parser.py

```diff
@@ -0,0 +1,61 @@
+import re
+import sys
+
+class EndpointSortStrategies(object):
```

Comment:
Pythonic way of doing this would be just to put the function in the top-level module namespace.